src/styles/blocks/card.scss

```diff
@@ -0,0 +1,91 @@
+.card {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  @extend %card-border;
+  font-family: "Roboto", Helvetica, sans-serif;
+  font-style: normal;
+  margin: 0 48px 46px 0;
```

Comment:
You couldn't use margins for a block. Use mix and move this code for example to `page__card`
 More detail in video.
[video, timing 4:15](https://mate.academy/learn/css-advanced/bem#/video/911)
![Screenshot 2022-07-21 at 20 13 04](https://user-images.githubusercontent.com/17098736/180273729-dc93ed10-4ce0-4c8f-8f42-c286abcb1ec9.png)